doc/using_pytorch.rst

```diff
@@ -566,11 +570,91 @@ The function should return a byte array of data serialized to content_type.
 The default implementation expects ``prediction`` to be a torch.Tensor and can serialize the result to JSON, CSV, or NPY.
 It accepts response content types of "application/json", "text/csv", and "application/x-npy".
 
-Working with Existing Model Data and Training Jobs
-==================================================
 
-Attach to existing training jobs
---------------------------------
+Bring your own model
+====================
+
+You can deploy a PyTorch model that you trained outside of SageMaker by using the ``PyTorchModel`` class.
+Typically, you save a PyTorch model as a file with extension ``.pt`` or ``.pth``.
+To do this, you need to:
+
+* Write an inference script.
+* Package the model artifacts into a tar.gz file.
+* Upload the tar.gz file to an S3 bucket.
+* Create the ``PyTorchModel`` object.
+
+Write an inference script
+-------------------------
+
+You must create an inference script that implements (at least) the ``predict_fn`` function that calls the loaded model to get a prediction.
```

Comment:
this contradicts https://github.com/aws/sagemaker-python-sdk/blob/31c8ff1ad5d1f4377d32262cfd1a378a3a1be71d/doc/using_pytorch.rst#serve-a-pytorch-model, which says that only `model_fn` is required (unless using EI)